src/panels/lovelace/cards/hui-plant-status-card.ts

```diff
@@ -29,10 +29,10 @@ import { LovelaceCard, LovelaceCardEditor } from "../types";
 import { PlantAttributeTarget, PlantStatusCardConfig } from "./types";
 
 const SENSOR_ICONS = {
-  moisture: mdiWater,
+  moisture: mdiWaterPercent,
   temperature: mdiThermometer,
   brightness: mdiWhiteBalanceSunny,
-  conductivity: mdiEmoticonPoop,
+  conductivity: mdiLeaf,
```

Comment:
I don't get this one, this is about the nutrients in the soil right?

Reply:
Thx, I like the idea. PR updated